WORKSPACE

```diff
@@ -39,9 +39,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # 3) Check if envoy_build_config/extensions_build_config.bzl is up-to-date.
 # Try to match it with the one in source/extensions and comment out unneeded extensions.
 
-ENVOY_SHA1 = "99706022206acb99bd8093c654fc84f6cd5a877e"  # 2020-04-06
+ENVOY_SHA1 = "74290ef76a76fbbf50f072dc33438791f93f68c7"  # 2020-05-23
```

Comment:
My pr https://github.com/envoyproxy/envoy/pull/11309 just merged.  Could you include it in?

Reply:
Ack